src/main/java/bisq/core/payment/PaymentModule.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.core.payment;
+
+import bisq.common.app.AppModule;
+import bisq.core.payment.validation.AssetProviderRegistry;
+import org.springframework.core.env.Environment;
+
+public class PaymentModule extends AppModule {
+
+    public PaymentModule(Environment environment) {
+        super(environment);
+    }
+
+    @Override
+    protected final void configure() {
+        bind(AssetProviderRegistry.class).toInstance(AssetProviderRegistry.getInstance());
```

Comment:
This will require a change in `bisq-desktop` to include PaymentModule or it will hit runtime exception.